code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -6954,7 +7280,7 @@
 					H.adjustToxLoss(3)
 				H.adjustToxLoss(0.3)
 
-/datum/reagent/ethanol/deadrumm/pintpointer
```

Comment:
Woah

Reply:
`deadrumm`

Reply:
I know, was surprised it stayed that way for so long

Reply:
there were a bunch of reagents defined twice too!

Reply:
\>deadrum**m**
god fuck